pyproject.toml

```diff
@@ -12,6 +12,8 @@ name = "napari-sphinx-theme"
 description = "Bootstrap-based Sphinx theme from the napari community"
 dynamic = ["version"]
 readme = "README.md"
+author = "napari team"
+author_email = napari-steering-council@googlegroups.com
```

Comment:
Does the email need to be in quotes @neuromusic? 

At any rate, I'm thinking maybe remove the email altogether, as it feels like an inappropriate place for requests about the theme... This repo is the right place I think...?

Reply:
good Q. probably.

I was just parroting this off the napari setup.cfg

Reply:
happy to drop the email